### PR TITLE
deps: update protobufjs and word-wrap deps

### PR DESCRIPTION
**Pull request by chingor13**
Ran `npm audit fix`

### CODE REVIEW

**Review by alexander-fenster — APPROVED** [review]

File: .github/workflows/finale.js [hunk 1]
```diff
@@ -25,7 +25,7 @@ async function main() {
   // 20 minutes * 60 seconds/min / 5 seconds per loop.
   const iterations = 20 * 60 / 5
   for (let i=0; i<iterations; i++) {
-    const url = `https://api.github.com/repos/${repo}/actions/runs/${runId}/jobs`;
+    const url = `https://api.github.com/repos/${repo}/actions/runs/${runId}/jobs?per_page=100`;
```
Comment (alexander-fenster) [inline .github/workflows/finale.js:28]: This seems unrelated - it may or may not be a part of this PR

Comment (chingor13) [reply .github/workflows/finale.js:28]: I needed that to make the CI pass. This PR needs to make sure more than 30 CI jobs succeed and the default page size was 30.

**Review by SurferJeffAtGoogle — APPROVED** [review]
